src/interceptors/ClientRequest/utils/createRequest.test.ts

```diff
@@ -83,3 +83,43 @@ it('creates a fetch Request with an empty string body', async () => {
   expect(request.url).toBe('https://api.github.com/')
   expect(request.body).toBe(null)
 })
+
+it('creates a fetch Request with an empty username', async () => {
```

Comment:
Looks like the names of these test suites are mixed up: this one is about empty password, and the bottom one is about empty username, isn't it?

Reply:
oopsie. fixed 😅

Reply:
Thank you!